src/pages/Guides/Guides.tsx

```diff
@@ -38,7 +42,9 @@ function Guides(props: GuidesProps) {
 
       <div className={classes.header}>
         <div className={classes.title}>Articles</div>
-        <a>See all</a>
+        <div className={classes.list}>
+          <Link  to={{pathname: 'Guides/Articles'}} >See all</Link>
+        </div>
       </div>
       <div className={classes.scroll}>
         {articles.map((key: any) => (
```

Comment:
Sorry one more small thing!! You did really well matching the guides thing-- but I think in the next line in`to={`${props.match.url}/${key}`}`, if we could adjust that to `to={`${props.match.url}/articles/${key}`}`to match the SeeAll version (and do a similar thing for videos and lessons)?